core/client/app/components/gh-profile-image.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    email: '',
+    size: 90,
+    hasUploadedImage: true,
```

Comment:
What does this do?  I don't see it being set anywhere so at least superficially, it appears as though it's always just `true`?